otter/metrics.py

```diff
@@ -165,8 +164,8 @@ def calc_total(group_metrics):
 
 
 @do
-def add_to_cloud_metrics(ttl, region, group_metrics, no_tenants, log=None,
-                         _print=False):
+def add_to_cloud_metrics(ttl, region, group_metrics, no_tenants, config,
```

Comment:
It'd also be nice if `no_tenants` could be renamed `num_tenants` or something, since "no" is a confusing abbreviation for "number".